ga4gh/cli.py

```diff
@@ -1496,6 +1498,29 @@ def configtest_main(parser=None):
 ##############################################################################
 
 
+def getNameFromPath(filePath):
+    """
+    Returns the filename of the specified path without its extensions.
+    This is usually how we derive the default name for a given object.
+    """
+    if len(filePath) == 0:
+        raise ValueError("Cannot have empty path for name")
+    fileName = os.path.split(filePath)[1]
+    # We need to handle things like .fa.gz, so we can't use
+    # os.path.splitext
+    ret = fileName.split(".")[0]
+    assert ret != ""
```

Comment:
Ahh, now I get you! To be honest, I think this is a corner case  we can ignore for now. This specific case (which is a problem in the current server) will go away because we're not trying to read in files for  each reference any more. I think we can safely say now that if the automatic name generation fails in these obscure cases then it's up to the user to step in and use the ``--name=GL000248.1`` command line option.

Feel free to open an issue on this if you disagree though.